src/utils/package-manager.ts

```diff
@@ -43,6 +44,32 @@ export const BUN: PackageManager = {
     });
   },
 };
+export const DENO: PackageManager = {
+  name: 'deno',
+  label: 'Deno',
+  installCommand: 'install',
+  buildCommand: 'deno task build',
+  runScriptCommand: 'deno task',
+  flags: '',
+  forceInstallFlag: '--force',
+  registry: 'npm',
+  detect: () =>
+    ['deno.lock'].some((lockfile) =>
+      fs.existsSync(path.join(process.cwd(), lockfile)),
+    ),
```

Comment:
This could be surrounded with a try/catch

Reply:
It looks like only other package managers that have a try catch block are the yarn ones, would you like me to add them to others while I'm here?

Reply:
Yes, that would be awesome. Thanks!